src/components/carbon-provider/carbon-provider.pw.tsx

```diff
@@ -0,0 +1,341 @@
+          "border-color",
+          color
+        );
+      });
+
+      test("Pill component and verify theme color", async ({ mount, page }) => {
+        await mount(<Pill>Foo</Pill>, { hooksConfig: { theme: `${theme}` } });
+
+        await expect(pillPreview(page)).toHaveCSS("border-color", color);
+      });
+
+      test("SplitButton component and verify theme color", async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <SplitButton text="Split button">
+            <Button>Button 1</Button>
+            <Button>Button 2</Button>
+            <Button>Button 3</Button>
+          </SplitButton>,
+          { hooksConfig: { theme: `${theme}` } }
+        );
+
+        await expect(buttonDataComponent(page)).toHaveCSS("color", color);
+        await expect(buttonDataComponent(page)).toHaveCSS(
+          "border-color",
+          color
+        );
+      });
+
+      test("Tabs component and verify theme color", async ({ mount, page }) => {
+        await mount(
+          <Tabs align="left" position="top">
+            <Tab tabId="tab-1" title="Tab 1" key="tab-1">
+              Content for tab 1
+            </Tab>
+          </Tabs>,
+          { hooksConfig: { theme: `${theme}` } }
+        );
+
+        const tabTitleSelectedIndicator = page.locator(
+          '[data-element="tab-selected-indicator"]'
+        );
+        await expect(tabTitleSelectedIndicator).toHaveCSS(
+          "box-shadow",
+          `${color} 0px -2px 0px 0px inset`
+        );
+      });
+    }
+  );
+});
+
+buildTestArray(commonColorsOnHover).forEach(([theme, color]) => {
+  test.describe(`should render components with ${theme} theme onHover`, () => {
+    test("Button component and verify theme color", async ({ mount, page }) => {
+      await mount(<Button>Small</Button>, {
+        hooksConfig: { theme: `${theme}` },
+      });
+
+      await buttonDataComponent(page).hover();
+      await expect(buttonDataComponent(page)).toHaveCSS(
+        "background-color",
+        color
+      );
+    });
+
+    test("Link component and verify theme color", async ({ mount, page }) => {
+      await mount(
+        <Link
+          href="https://carbon.sage.com"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          This is a link
+        </Link>,
+        { hooksConfig: { theme: `${theme}` } }
+      );
+
+      await linkComponent(page).hover();
+      await expect(linkComponent(page)).toHaveCSS("color", color);
+    });
+
+    test("SplitButton component and verify theme color", async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <SplitButton text="Split button">
+          <Button>Button 1</Button>
+          <Button>Button 2</Button>
+          <Button>Button 3</Button>
+        </SplitButton>,
+        { hooksConfig: { theme: `${theme}` } }
+      );
+
+      const toggleButton = await page.locator('[data-element="toggle-button"]');
```

Comment:
**issue**: unnecessary `await` here

Reply:
You know my philosophy.  `If you're stuck in a strait, add an await`